src/frontend/components/LiveVideoInformationBar/LiveControlButtons/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+
+import { theme } from 'utils/theme/theme';
+
+import { StudentShareDocButton } from './StudentShareDocButton';
+import { StudentShowViewersButton } from './StudentShowViewersButton';
+import { StudentShowChatButton } from './StudentShowChatButton';
+import { StudentJoinDiscussionButton } from './StudentJoinDiscussionButton';
+import { StudentQuitDiscussionButton } from './StudentQuitDiscussionButton';
+
+const LiveButtonsWrapper = styled(Box)`
+  div {
+    position: relative;
+
+    span {
+      position: absolute;
```

Comment:
ditto please use alphabetical order

Reply:
css rules should also be sorted in alphabetic order ?